src/components/FormMaxInputToggleButton.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+import { ButtonShape, ButtonSize } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+
+import { formMixins } from '@/styles/formMixins';
+
+import { Icon, IconName } from './Icon';
+import { ToggleButton } from './ToggleButton';
+
+type ElementProps = {
+  size?: ButtonSize;
+  isInputEmpty: boolean;
+  isLoading: boolean;
+  onPressedChange: (isPressed: boolean) => void;
+};
+
+export const FormMaxInputToggleButton = ({
```

Comment:
should we reuse this button in other forms too? transfers/deposit etc? :D

Reply:
yess can file follow up to do so!